visidata/path.py

```diff
@@ -130,13 +130,13 @@ def read(self, size=-1):
         r = self.fp_orig_read(size)
         if self.prog:
             if r:
-                self.prog.addProgress(len(r))
+                self.prog.addProgress(len(r.encode(self.encoding)) if self.encoding else len(r))
```

Comment:
Unfortunately this re-encodes every decoded line, which I think will have a significant impact on performance.  Is there another way to do this?

Reply:
I'm okay with estimation, doing it every 100 or 1000 calls to `read()`.